src/utils/object.js

```diff
@@ -44,3 +44,25 @@ export function map(opt_initial) {
 export function hasOwn(obj, key) {
   return hasOwn_.call(obj, key);
 }
+
+/**
+ * Returns the value of the property referenced by dot-separated keys.
+ * e.g.
+ * `getPath({a: {b: [{c: 2}]}}, 'a.b[0].c') === 2`
+ *
+ * @param {*} obj a map-like value
+ * @param {string} path a dot-separated list of keys to reference a value
+ * @return {*}
+ */
+export function getPath(obj, path) {
```

Comment:
@cvializ Is this method significantly different from https://github.com/ampproject/amphtml/blob/master/src/json.js#L84 ?

If no, please file a bug to reconcile with P1.